vfio-ubind/src/main.rs

```diff
@@ -82,19 +77,18 @@ fn main() {
 		println!("No signs of an IOMMU. Check your hardware and/or linux cmdline parameters.");
 		println!("Use intel_iommu=on or iommu=pt iommu=1");
 		info!("File {} didn't exist", dev_iommu.display());
-		return;
+		panic!();
```

Comment:
Please include the error message into the `panic` instead of printing it normally beforehand.